src/ui/bounding_polygon_plugin.cpp

```diff
@@ -401,6 +399,9 @@ bool Bounding_Polygon_Menu::post_draw() {
 
     ImGui::SameLine();
     ImVec2 cursor_pos = ImGui::GetCursorScreenPos();
+#ifdef __APPLE__
+    cursor_pos.x *= (1.0+keyframe_nudge_amount);
+#endif
```

Comment:
What is this code doing? Is it necesarry to compile the app?

Reply:
It's meant to correct an issue with the "Next KF" button, which is truncated/cut off with the normal code. I can get rid of it if you don't think it's really a big deal at this point.